OWNERS_ALIASES

```diff
@@ -193,20 +193,20 @@ aliases:
     - shurup
   sig-docs-pl-owners: # Admins for Polish content
     - mfilocha
-    - nvtkaszpir
+    # nvtkaszpir
```

Comment:
This will leave Polish unviable as a localization (only 1 owner)

Reply:
@sftim - @nvtkaszpir confirmed his readiness to help, see PR's comments above.